docs/user-guide/gpudeduplication.rst

```diff
@@ -195,6 +398,5 @@ steps (all scripts are included in the :code:`nemo_curator/scripts/` subdirector
                    --log-dir ./
                    # --scheduler-file /path/to/file.json
 
-In addition to the scripts, there are examples in the `examples` directory that showcase using the python module
-directly in your own code. It also has examples on how to remove documents from the corpus using the list of duplicate IDs generated from exact or fuzzy
-deduplication.
+All subsequent steps starting with :ref:`Buckets <fuzzydup_lsh>` can be executed on all the data
```

Comment:
```suggestion
All subsequent steps, starting with :ref:`Buckets <fuzzydup_lsh>`, can be executed on all the data
```
Where does the link point to?

Reply:
Points back to the `2. Buckets (Minhash Buckets)` section of the CLI.